Detections/AuditLogs/AzureADRoleManagementPermissionGrant.yaml

```diff
@@ -0,0 +1,55 @@
+id: 1ff56009-db01-4615-8211-d4fda21da02d
+name: Azure AD Role Management Permission Grant
+description: |
+  'Identifies when the Microsoft Graph RoleManagement.ReadWrite.Directory (Delegated or Application) permission is granted to a service principal.
+  This permission allows an application to read and manage the role-based access control (RBAC) settings for your company's directory.
+  An adversary could use this permission to add an Azure AD object to an Admin directory role.
+  Ref : https://docs.microsoft.com/en-us/graph/permissions-reference#role-management-permissions
```

Comment:
Remove the en-us from the reference links, this is causing the validation failure.

Reply:
Done. Thank you Shain!